app/blog/posts/good-python-code.mdx

```diff
@@ -0,0 +1,333 @@
+---
+title: "Everything I've learnt about writing good Python code"
+publishedAt: "2024-04-17"
+description: Speedrun your way to becoming a good python developer and don't make the same mistakes I did
+---
+
+I've always used python for my interviews but it wasn't until about 6 months ago that I started to use python for all sorts of projects - from simple machine learning scripts to full blown applications. In the proccess, I've learnt a lot about what constitutes good python code and I hope that you'll find the tips in this article useful in your own projects.
```

Comment:
can definitely use a stronger hook. 
Today I'm going to share with you end tips on how to write better code that I learn from so on the thing.

Reply:
I changed it to 

> In the past 6 months, I've 10xed the amount of python code I've written. In this article, I'll show you a few easy actionable tips to write better and more maintanable code. I've been lucky enough to have Jason (@jxnlco on twitter) review a good chunk of my code and I've found that these few things have made a massive difference in my code quality.